openupgrade_scripts/scripts/hr_expense/15.0.2.0/end-migration.py

```diff
@@ -6,6 +6,27 @@ def recompute_total_amount_company(env):
     expense_to_recompute._compute_total_amount_company()
 
 
+def update_expense_unit_amount(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE hr_expense
+        SET unit_amount = 0.0
+        WHERE id IN (
+            SELECT he.id
+            FROM hr_expense he
+            JOIN product_product pp ON he.product_id = pp.id
+            LEFT JOIN ir_property ip ON ip.name = 'standard_price'
+                AND ip.res_id = CONCAT('product.product,', pp.id)
+                AND he.company_id = ip.company_id
+            WHERE  he.unit_amount != 0.0
```

Comment:
Như này nghĩa là toàn bộ expense mà có sản phẩm là standard price thì đưa hết unit_amount về 0 hết à chú?

Reply:
@hoangtiendung070797  review đoạn SQL này nhé.

Reply:
@vietphuong10 ok Phương

Reply:
Vấn đề này xử lý ở End-Migration cho novamed nhé @Khoa-Jocelyn